.rubocop.yml

```diff
@@ -2,6 +2,8 @@ MethodLength:
   Enabled: false
 ClassLength:
   Enabled: false
+Documentation:
+    Enabled: false
```

Comment:
can you fix indent here to 2 spaces?  bad style in a style-disabling commit is too ironic ;-)